lib/formatters.js

```diff
@@ -77,7 +81,7 @@ var formatInputBool = function (value) {
 /// Values are multiplied by 2^m and encoded as integers
 /// @returns byte representation of real
 var formatInputReal = function (value) {
-    return formatInputInt(new BigNumber(value).times(new BigNumber(2).pow(128))); 
+    return formatInputInt(new BigNumber(value).mul(new BigNumber(2).toRed(BigNumber.red('k256')).redPow(new BigNumber(128)).fromRed())); 
```

Comment:
is k256 ok for the max int? Do you have max that ethereum.js can handle defined somewhere?